code/game/objects/items/storage/reagent_pouch.dm

```diff
@@ -0,0 +1,167 @@
+//Reagent Canister pouch. Including the canister inside the pouch, as well as the pouch item.
+
+/obj/item/reagent_containers/glass/reagent_canister // See the Reagent Canister Pouch, this is just the container
+	name = "pressurized reagent container"
+	desc = "A pressurized container. The inner part of a pressurized reagent canister pouch. Too large to fit in anything but the pouch it comes with."
+	icon = 'icons/Marine/marine-pouches.dmi'
+	icon_state = "r_canister"
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/inhands/equipment/tanks_left.dmi',
+		slot_r_hand_str = 'icons/mob/inhands/equipment/tanks_right.dmi',
+	)
+	item_state = "anesthetic"
+	possible_transfer_amounts = null
+	volume = 1200 //The equivalent of 5 pill bottles worth of BKTT
+	w_class = WEIGHT_CLASS_BULKY
+
+/obj/item/reagent_containers/glass/reagent_canister/examine(mob/user)
+	. = ..()
+	. += get_examine_info(user)
+
+///Used on examine for properly skilled people to see contents.
+/obj/item/reagent_containers/glass/reagent_canister/proc/get_examine_info(mob/user)
+	if(isxeno(user))
+		return
+	if(!(user.skills.getRating(SKILL_MEDICAL) >= SKILL_MEDICAL_NOVICE)) //Failed skill check
+		return "You don't know what's in it."
+	if(isnull(reagents.total_volume))
+		return span_notice("[src] is empty!")
```

Comment:
Wouldn't you want it to say it's empty if it's 0 too? Just do `!reagents.total_volume`